docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -17,6 +17,22 @@ There you can find links to upgrade notes for other versions too.
 - fix the typo in Twig template `@ShopsysShop/Front/Content/Category/panel.html.twig` ([#1043](https://github.com/shopsys/shopsys/pull/1043))
     - `categoriyWithLazyLoadedVisibleChildren` ⟶ `categoryWithLazyLoadedVisibleChildren`
 - create an empty file `app/Resources/.gitkeep` to prepare a folder for [your overwritten templates](/docs/cookbook/modifying-a-template-in-administration.md) ([#1073](https://github.com/shopsys/shopsys/pull/1073))
+- fix `FilterQueryTest` to use ElasticSearch index prefix properly ([#1082](https://github.com/shopsys/shopsys/pull/1082))
+    ```diff
+    - 16:   private const ELASTICSEARCH_INDEX = 'product1';
+    + 16:   private const ELASTICSEARCH_INDEX = 'product';
+    ...
```

Comment:
```suggestion
    ```
    ```diff
```
Maybe it would make sense removing the line numbers and divide it into two diffs like this instead.